Formula/jdupes.rb

```diff
@@ -20,7 +20,20 @@ class Jdupes < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "8df5e09093977d45b08da78175dd18e8f221734d34c95cca11be1fec6beca13d"
   end
 
+  resource "libjodycode" do
+    url "https://github.com/jbruchon/libjodycode/archive/refs/tags/v1.0.tar.gz"
+    sha256 "3849c7a76c46687eafcff8db37477ce31662ac7a0d88cbd7495755b0f9859280"
+
+    patch :DATA
+  end
```

Comment:
we can also make a new formula for it.

Reply:
The repo does not meet the notability requirements. Of course that's a requirement we imposed ourselves but I figured I'd mention it.